app/models/quota_order_number.rb

```diff
@@ -9,6 +9,11 @@ class QuotaOrderNumber < Sequel::Model
   end
 
   dataset_module do
+    def by_order_number(order_number)
```

Comment:
maybe `for_order_number` - **by** would imply sorting rather then filtering

or possibly `find_by_order_number` ? Though that might imply returning a single record rather than an array

Reply:
I'm just going with a common idiom in the backend, here (by_code, by_producline_suffix, etc)